help/help_src/2. Business Reference Model.md

```diff
@@ -1,5 +1,25 @@
 # Business Reference Model
 
-Here is stuff about the Business Reference Model, with an example of including an image
+## Architecture Portal – Business Reference Model 
 
-<img src="img/screenshot-brm.png" />
```

Comment:
Is this screenshot still required?

Reply:
For now yup

Reply:
is it referenced elsewhere now? if so I may have missed it

Reply:
Ill remove it :)

Reply:
its referenced on the main readme file of this repo, maybe just leave it. I was only checking if the reference to the screenshot on this page was still needed